### ISSUE TITLE
Semantic Constrastive Mapping(SCM) loss is not consistent with the paper 

### ISSUE THREAD

**Issue by zzzzzzzzzzzx** [qy-feng/margin-openset#4]
Hi,
According to the paper, the Semantic Constrastive Mapping(SCM) loss has parameter ρ and w. like below
![22](https://user-images.githubusercontent.com/44466906/136881629-8c9c0abb-f08d-4f2b-8c63-8b662d47522e.PNG)
However, in this code, i cannot find these two parameters ρ and w. The SCM loss = D_k / n_k - D_u / n_u, without any weight. I am wondering whether part of the loss code is missing. Could u pls provide me this part of code? And i don't understand the ρ represent what cosine similarity.  Thanks!
![捕获](https://user-images.githubusercontent.com/44466906/136881692-5e7c898b-c396-4575-8800-0c3aaac3a31d.PNG)
